gh-gl-sync/SpackCIBridge.py

```diff
@@ -236,7 +257,7 @@ def update_refspecs_for_tags(self, tags, open_refspecs, fetch_refspecs):
     def fetch_github_branches(self, fetch_refspecs):
         """Perform `git fetch` for a given list of refspecs."""
         print("Fetching GitHub refs for open PRs")
-        fetch_args = ["git", "fetch", "-q", "github"] + fetch_refspecs
+        fetch_args = ["git", "fetch", "-q", "--depth=1", "github"] + fetch_refspecs
```

Comment:
N.B.  I don't *think* this shallow fetch will mess anything up, but I wanted your opinion on this change before we merge.